lua/compile-mode/init.lua

```diff
@@ -248,7 +248,10 @@ local runcommand = a.void(
 		local prev_win = vim.api.nvim_get_current_win()
 		local bufnr = utils.split_unless_open({ fname = config.buffer_name }, param.smods or {}, param.count)
 		utils.wait()
+
+		local output_win = vim.api.nvim_get_current_win()
 		vim.api.nvim_set_current_win(prev_win)
+
```

Comment:
Seems like we don't use `output_win`...

```suggestion
		vim.api.nvim_set_current_win(prev_win)
```